gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/bottomsheet/GravatarQuickEditorBottomSheet.kt

```diff
@@ -0,0 +1,89 @@
+package com.gravatar.quickeditor.ui.editor.bottomsheet
+
+import android.util.Log
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.material3.SheetState
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import com.gravatar.quickeditor.R
+import com.gravatar.quickeditor.ui.editor.GravatarQuickEditorParams
+import com.gravatar.quickeditor.ui.editor.GravatarQuickEditorPage
+import com.gravatar.ui.GravatarTheme
+import kotlinx.coroutines.launch
+
+/**
+ * ModalBottomSheet component for the Gravatar Quick Editor.
+ *
+ * The bottom sheet is configured to take 70% of the screen height and skips the partially expanded state.
+ *
+ * @param gravatarQuickEditorParams The Gravatar Quick Editor parameters.
+ * @param onDismiss The callback for when the bottom sheet is dismissed.
+ * @param modalBottomSheetState The state of the bottom sheet.
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+public fun GravatarQuickEditorBottomSheet(
+    gravatarQuickEditorParams: GravatarQuickEditorParams,
+    onDismiss: () -> Unit,
```

Comment:
What other callback we might want to provide here? 
We will likely need something like `onProfileUpdated`. I wonder if we want to communicate the OAuth errors here as well or do we want to keep it within the SDK and handle ourselves fully?

Reply:
>Yeah. Do we want to be less generic for the moment? For example, onAvatarUpdated, and if we later add the possibility of modifying the profile, we'll add a different callback.


If we decide to go for the scopes I was thinking about something like `onProfileUpdated(ProfilePayload)` -> The `ProfilePayload` would then correspond to the scope selected.

Having a simple `onProfileUpdated` and returning a full profile would be simpler for third-party devs to handle I guess, but we won't have that. We might not even have access to the full profile. 

>Good question. I'm trying to use the third-party developer hat, and I guess it could be useful to know, at least, that maybe I decide to show something in the UI? However, I wouldn't be too specific with the error details.

Yeah, we probably need at least one error to propagate when the OAuth fails - as this could be a config issue.